ReactiveArchitecture/Classes/core/extension/ConnectableObservableExtension.swift

```diff
@@ -27,7 +27,7 @@ import RxSwift
 extension ConnectableObservableType {
     func autoconnect() -> Observable<E> {
         return Observable.create { observer in
-            return self.do(onSubscribe: {
+            self.do(onSubscribe: {
```

Comment:
Prefer implicit return of explicit

Reply:
I bet Aziz Ansar would have loved explicit LOL!

I had no idea swift did this. However as a learning tool I'm probably going to leave it. Less WTF moments.